src/components/annual-report/report-data/section4/section4Calculations.js

```diff
@@ -0,0 +1,158 @@
+import { getRoundedValue } from '../utils/mathService.js'
+import { mapSectionFourResponse } from '../utils/utils.js'
+
+const OVERALL_RESPONSE_NAME = 'data_over_all_org_units'
+const BY_LEVEL_RESPONSE_NAME = 'data_detail_by_level'
+const DENOMINATOR_RELATIONS_MAP = 'denominatorRelationsMap'
+
+const getVal = ({ response, dx, ou, pe }) => {
+    return response?.[dx]?.[ou]?.[pe]
+}
+
+const calculateSection4a = ({
+    formattedResponseOverall,
+    denominatorRelations,
+    currentPeriod,
+    overallOrgUnit,
+}) => {
+    const section4a = []
+    for (const rel of denominatorRelations) {
+        if (rel.type !== 'un') {
+            break
+        }
```

Comment:
Are `un`-type relations necessarily first in the array? If not, they might be missed -- using `continue` here instead of `break` would fix that

Not related to the code here, but I'm also curious -- is there ever more than 1 `un`-type relation? 🤔

Reply:
Yikes 🙈. That should be `continue`. Thanks!